app/views/shared/_navigation.html.erb

```diff
@@ -7,6 +7,13 @@
   </div>
 
   <div class="header_menu">
+    <section id="user-info">
+      <div class="prof-link-container">
```

Comment:
I don't think shaving off a few letters saves you much here but can add a bit of confusion, I think the class should just be `profile-link-container`

Reply:
This is the same class name used in MMT